src/libs/antares/study/parts/short-term-storage/series.cpp

```diff
@@ -115,6 +114,45 @@ void Series::fillDefaultSeriesIfEmpty()
     fillIfEmpty(upperRuleCurve, 1.0);
 }
 
+bool Series::saveToFolder(const std::string& folder) const
+{
+    logs.debug() << "Saving series into folder: " << folder;
+#define SEP Yuni::IO::Separator
+    if (!Yuni::IO::Directory::Create(folder))
+    {
+        logs.warning() << "Couldn't create dir: " << folder;
+        return false;
+    }
+
+    bool ret = true;
+    ret = writeVectorToFile(folder + SEP + "PMAX-injection.txt", maxInjectionModulation) && ret;
+    ret = writeVectorToFile(folder + SEP + "PMAX-withdrawal.txt", maxWithdrawalModulation) && ret;
+    ret = writeVectorToFile(folder + SEP + "inflows.txt", inflows) && ret;
+    ret = writeVectorToFile(folder + SEP + "lower-rule-curve.txt", lowerRuleCurve) && ret;
+    ret = writeVectorToFile(folder + SEP + "upper-rule-curve.txt", upperRuleCurve) && ret;
+#undef SEP
+    return ret;
+}
+
+bool writeVectorToFile(const std::string& path, const std::vector<double>& vect)
+{
+    try
+    {
+        std::ofstream fout(path);
+        fout << std::setprecision(14);
+
+        for (const auto& x : vect)
+            fout << x << std::endl;
```

Comment:
```suggestion
            fout << x << '\n';
```
See https://stackoverflow.com/a/8311177